fullnode/prod/0.6.0/config/config.toml

```diff
@@ -0,0 +1,348 @@
+addr_book_file = "config/addrbook.json"
+
+# Set true for strict address routability rules
+# Set false for private or local networks
+addr_book_strict = false
+
+# Maximum number of inbound peers
+max_num_inbound_peers = 40
+
+# Maximum number of outbound peers to connect to, excluding persistent peers
+max_num_outbound_peers = 10
+
+# Time to wait before flushing messages out on the connection
+flush_throttle_timeout = "10ms"
+
+# Maximum size of a message packet payload, in bytes
+max_packet_msg_payload_size = 10485760
+
+# Maximum num of keys a state sync request ask for
+keys_per_request = 1500
+
+# Rate at which packets can be sent, in bytes/second
+send_rate = 52428800
+
+# Rate at which packets can be received, in bytes/second
+recv_rate = 102428800
+
+# Interval to send pings
+ping_interval = "10m0s"
+
+# Maximum wait time for pongs
+pong_timeout = "450s"
+
+# Set true to enable the peer-exchange reactor
+pex = true
+
+# Seed mode, in which node constantly crawls the network and looks for
+# peers. If another node asks it for addresses, it responds and disconnects.
+#
+# Does not work if the peer-exchange reactor is disabled.
+seed_mode = false
+
+# Comma separated list of peer IDs to keep private (will not be gossiped to other peers)
+private_peer_ids = ""
+
+# Toggle to disable guard against peers connecting from the same ip.
+allow_duplicate_ip = true
+
+# Peer connection configuration.
+handshake_timeout = "20s"
+dial_timeout = "3s"
+
+##### dbcache configuration options #####
+[dbcache]
+# OpenFilesCacheCapacity defines the capacity of the open files caching.
+open_files_cache_capacity = 1024
+
+# BlockCacheCapacity defines the capacity of the 'sorted table' block caching.
+block_cache_capacity = 8388608
+
+# WriteBuffer defines maximum size of a 'memdb' before flushed to 'sorted table'.
+write_buffer = 4194304
+
+# Filter defines an 'effective filter' to use. An 'effective filter'
+# if defined will be used to generate per-table filter block.
+# Greater than 0 would creates a new initialized bloom filter for given bitsPerKey.
+bits_per_key = 10
+
+##### mempool configuration options #####
+[mempool]
+
+recheck = true
+broadcast = true
+wal_dir = ""
+
+# Maximum number of transactions in the mempool
+size = 5000
+
+# Limit the total size of all txs in the mempool.
+# This only accounts for raw transactions (e.g. given 1MB transactions and
+# max_txs_bytes=5MB, mempool will only accept 5 transactions).
+max_txs_bytes = 1073741824
+
+# Size of the cache (used to filter transactions we saw earlier) in transactions
+cache_size = 10000
+
+##### consensus configuration options #####
+[consensus]
+
+wal_file = "data/cs.wal/wal"
+
+timeout_propose = "3s"
+timeout_propose_delta = "500ms"
+timeout_prevote = "1s"
+timeout_prevote_delta = "500ms"
+timeout_precommit = "1s"
+timeout_precommit_delta = "500ms"
+timeout_commit = "1s"
+
+# Make progress as soon as we have all the precommits (as if TimeoutCommit = 0)
+skip_timeout_commit = false
```

Comment:
THIS SHOULD BE TRUE
https://github.com/binance-chain/node-binary/blob/master/fullnode/prod/0.5.10/config/config.toml#L226